src/codegen/enums.rs

```diff
@@ -354,7 +353,7 @@ impl FromGlib<{sys_crate_name}::{ffi_name}> for {name} {{
         writeln!(
             w,
             "impl<'a> FromValueOptional<'a> for {name} {{
-    unsafe fn from_value_optional(value: &Value) -> Option<Self> {{
+    unsafe fn from_value_optional(value: &glib::Value) -> Option<Self> {{
```

Comment:
This needs to use `use_glib_type("Value")` and that would also fix the CI (partially at least).

Same in the other places below